pkgs/applications/misc/gramps/default.nix

```diff
@@ -1,26 +1,27 @@
-{ lib
-, fetchFromGitHub
-, gtk3
-, pythonPackages
-, glibcLocales
-, intltool
-, gexiv2
-, pango
-, gobject-introspection
-, wrapGAppsHook3
-, gettext
+{
+  lib,
+  fetchFromGitHub,
+  gtk3,
+  pythonPackages,
+  glibcLocales,
+  intltool,
+  gexiv2,
+  pango,
+  gobject-introspection,
+  wrapGAppsHook3,
+  gettext,
   # Optional packages:
-, enableOSM ? true
-, osm-gps-map
-, glib-networking
-, enableGraphviz ? true
-, graphviz
-, enableGhostscript ? true
-, ghostscript
+  enableOSM ? true,
+  osm-gps-map,
+  glib-networking,
+  enableGraphviz ? true,
+  graphviz,
+  enableGhostscript ? true,
+  ghostscript,
```

Comment:
Why the change to the formatting?

If you really want to use this formatting, use `nixfmt-rfc-style` on the entire file, but I think this is unnecessary for this PR.

This suggestion also applies to all other files that had formatting changes.

Reply:
Thanks will do so, I wasn't aware of the tool name and `nixfmt` currently does not do a good job I believe.

Reply:
I would recommend to just drop this commit form this PR

Reply:
I don't understand why or the reasoning behind your comment @SuperSandro2000